documentation/src/main/asciidoc/topics/con_eviction_overview.adoc

```diff
@@ -1,5 +1,15 @@
 [id='how_eviction_works-{context}']
 = How Eviction Works
+{brandname} eviction removes entries from the data container to make space when
+adding new entries.
+
+[IMPORTANT]
+====
+To prevent data loss, you should always configure a persistent cache store if
+you enable eviction. Use passivation so that {brandname} writes to persistent
```

Comment:
I don't think we need to mention passivation here. Makes it sound like that is how I should fix this.